CommonTools/PileupAlgos/python/Puppi_cff.py

```diff
@@ -31,6 +31,7 @@
                        UseDeltaZCut   = cms.bool(True),
                        DeltaZCut      = cms.double(0.3),
 		       PtMaxCharged   = cms.double(0.),
+		       EtaMaxCharged   = cms.double(0.),
```

Comment:
Please don't forger to update also here, as this config will overwrite the setting defined in the ```fillDescriptions``` methods

Reply:
all done